fvm/meter/weighted/meter.go

```diff
@@ -187,10 +306,10 @@ func (m *Meter) MemoryIntensities() interfaceMeter.MeteredMemoryIntensities {
 
 // TotalMemoryUsed returns the total memory used
 func (m *Meter) TotalMemoryUsed() uint {
-	return uint(m.memoryUsed >> MeterInternalPrecisionBytes)
```

Comment:
I think there is one more of these in the merge function. Also maybe we can rename the `MeterInternalPrecisionBytes` to `MeterExecutionInternalPrecisionBytes` or similar.